<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartIDDialog</class>
 <widget class="QDialog" name="SmartIDDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Smart-ID</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
font-family: Roboto, Helvetica;
font-size: 14px;
}
#SmartIDDialog {
background-color: #FFFFFF;
border-radius: 4px;
}
#label {
color: #003168;
font-size: 20px;
font-weight: 700;
}
#labelCode, #labelCountry {
color: #07142A;
}
#errorCode, #errorCountry {
color: #AD2A45;
}
QLineEdit, QComboBox {
padding: 10px 14px;
border: 1px solid #C4CBD8;
border-radius: 4px;
color: #07142A;
background-color: white;
placeholder-text-color: #607496;
font-size: 16px;
}
QComboBox QWidget#popup {
background-color: transparent;
}
QComboBox QWidget#content {
border: 1px solid #C4CBD8;
border-radius: 4px;
background-color: #FFFFFF;
}
QComboBox QPushButton {
margin: 3px;
padding: 0px 12px 0px 4px;
border: 0px;
color: #07142A;
text-align: left;
font-weight: normal;
font-size: 16px;
qproperty-iconSize: 14px 9px;
qproperty-layoutDirection: RightToLeft;
}
QComboBox QPushButton#selected {
qproperty-icon: url(:/images/arrow_up.svg);
}
QComboBox QPushButton:hover#selected {
qproperty-icon: url(:/images/arrow_up_white.svg);
}
QComboBox::drop-down {
background-color: #FFFFFF;
width: 25px;
}
QComboBox::down-arrow {
image: url(:/images/arrow_down.svg);
}
QComboBox::down-arrow:on {
top: 1px;
left: 1px;
}
QCheckBox {
spacing: 8px;
color: #07142A;
border: none; /*Workaround for right padding*/
}
QCheckBox:disabled {
color: #C4CBD8;
}
QCheckBox::indicator {
width: 16px;
height: 16px;
}
QCheckBox::indicator:unchecked {
image: url(:/images/icon_checkbox.svg);
}
QCheckBox::indicator:unchecked:hover {
image: url(:/images/icon_checkbox_active.svg);
}
QCheckBox::indicator:unchecked:disabled {
image: url(:/images/icon_checkbox_disabled.svg);
}
QCheckBox::indicator:checked {
image: url(:/images/icon_checkbox_check.svg);
}
QCheckBox::indicator:checked:hover {
image: url(:/images/icon_checkbox_check_active.svg);
}
QCheckBox::indicator:checked:disabled {
image: url(:/images/icon_checkbox_check_disabled.svg);
}
QPushButton {
padding: 12px 12px;
border-radius: 4px;
border: 1px solid #AD2A45;
color: #AD2A45;
font-weight: 700;
}
QPushButton:hover {
background-color: #F5EBED;
}
QPushButton:pressed {
background-color: #E1C1C6;
}
QPushButton:default {
color: #ffffff;
border-color: #2F70B6;
background-color: #2F70B6;
}
QPushButton:default:hover {
background-color: #2B66A6;
}
QPushButton:default:pressed {
background-color: #215081;
}
QPushButton:default:disabled {
background-color: #2F70B6;
}</string>
  </property>
  <layout class="QVBoxLayout" name="SmartIDDialogLayout">
   <property name="spacing">
    <number>40</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>32</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Enter your personal code to sign with Smart-ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutContent">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="layoutCountry">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCountry">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Country</string>
         </property>
         <property name="buddy">
          <cstring>idCountry</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ComboBox" name="idCountry">
         <item>
          <property name="text">
           <string>Estonia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lithuania</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Latvia</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutCode">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Personal code</string>
         </property>
         <property name="buddy">
          <cstring>idCode</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LineEdit" name="idCode">
         <property name="placeholderText">
          <string notr="true">47101010033</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorCode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cbRemember">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Remember me</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sign">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Sign</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/ComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
